### PR TITLE
Add RStudio user guide documentation

### PR DESCRIPTION
**Pull request by atheo89**
Related to: https://issues.redhat.com/browse/RHOAIENG-1851

## Description
You may find the readable link here -> [https://github.com/atheo89/notebooks/blob/rstudi-userguide/docs/rstudio-server-user-guide.md](https://github.com/atheo89/notebooks/blob/rstudi-userguide/docs/rstudio-server-user-guide.md )

## Merge criteria:

- [ ] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

**Review by harshad16 — APPROVED** [review]
/lgtm
/approve

thanks :100:

File: docs/rstudio-server-user-guide.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+### Launch RStudio Server
+
+To launch RStudio Server, there are two methods available:
+
+The first method involves accessing the Notebooks workbench spawner page:
+
+1.  Navigate to Applications and click on Enabled.
+2.  Locate the JupyterHub card and click Launch.
+3.  A new browser tab will open, presenting several notebooks. Select RStudio Server Workbench.
```
Comment (jstourac) [inline docs/rstudio-server-user-guide.md:17]: Hmm, I think that this page opened to me in the very same browser tab when  I accessed it last time :thinking:

File: docs/rstudio-server-user-guide.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+The first method involves accessing the Notebooks workbench spawner page:
+
+1.  Navigate to Applications and click on Enabled.
+2.  Locate the JupyterHub card and click Launch.
+3.  A new browser tab will open, presenting several notebooks. Select RStudio Server Workbench.  
+      
+The second method utilizes Data Science Projects:
+
+1.  Click the blue button labeled "Create data science project."
```
Comment (jstourac) [inline docs/rstudio-server-user-guide.md:21]:
```suggestion
1.  Navigate to Data Science Projects page
2.  Click the blue button labeled "Create data science project."
```

**Review by jstourac — APPROVED** [review]
LGTM, just two minor comments - rather optional.

**Pull request merged**